Documentation/GeneralConventions/ReviewPolicy.rst

```diff
@@ -0,0 +1,201 @@
+.. include:: /Includes.rst.txt
+.. index:: ! Review policy
+.. _review-policy:
```

Comment:
Can we add an empty row between directives and header names?

Reply:
We can, technically... should we? This should be put into the guide too if this is a "rule" (haven't checked)

Reply:
I  dont think empty lines are needed here. And I think we should already follow the review guideline. CGL issues can be done in a follow up